src/pyspiffe/workloadapi/default_workload_api_client.py

```diff
@@ -279,10 +279,13 @@ def fetch_jwt_svid(
         if not audiences:
             raise ArgumentError('Parameter audiences cannot be empty')
 
+        if subject is not None:
+            subject = str(subject)
+
```

Comment:
We are getting this error: `src/pyspiffe/workloadapi/default_workload_api_client.py:283: error: Incompatible types in assignment (expression has type "str", variable has type "Optional[SpiffeId]")  [assignment]`

```suggestion
        subject = str(subject) if subject is not None else ''
```